README.md

```diff
@@ -74,4 +74,29 @@ void main() async {
 
   await db.close();
 }
-```
\ No newline at end of file
+```
+
+# Web
+
+Web support is provided by the [Drift](https://github.com/powersync-ja/drift/pull/1) library. Detailed instructions for compatibility and setup are listed in the link.
+
+Web support requires Sqlite3 WASM and Drift worker Javascript files to be accessible via configurable URIs.
+
+Default URIs are shown in the example below. URIs only need to be specified if they differ from default values.
+
+Watched queries and table change notifications are only supported when using a custom Drift worker which is compiled by linking
+https://github.com/powersync-ja/drift/pull/1. 
```

Comment:
This looks outdated since we're not depending on Drift anymore

Reply:
This has been updated.